### ISSUE TITLE
Issues with fontkit and Roboto Thin

### ISSUE THREAD

**Issue by DHTMLGoodies** [pcwalton/font-kit#81]
We're experiencing some strange issues when using fontkit and the Google font "Roboto-Thin"

When working with this font locally(localhost - http), everything works fine, but once we deploy our app to a production server(https), i.e. minimize, package(zip) and upload, the letter "a" in the Roboto-Thin font is gone. 

I have done some console.log of the Fontkit.font object returned from this font. The letter "a" has acii code 97. If I understand the fontkit.font properly, I can see that characterSet entry 68 refers to 97. Then, when I look into the _glyphs array, I can see that the entry 68 has codePoints set to [97] on my localhost. However, on the https production server, the same _glyphs entry has codePoints set to an empty array(codePoints: []).  I assume this is the reason why we can't see the letter "a" there. 

In summary: 
- I'm parsing the same Roboto-Thin.ttf file on my localhost and on the production server(xhr request to same url).- the font can be downloaded from Google fonts.
- On my localhost, I get correct Font data back from fontkit
- On a https production server, I get different font data for the same font where codePoints for _glyphs[68] is empty. 

It's a very strange problem. Have you experienced this before, and do you have any tip on what the cause could be?

Thanks

---
Alf

**Comment by RazrFalcon** [comment]
Can you provide an md5sum for this font file or the file itself?

**Comment by DHTMLGoodies** [comment]
Thanks for your swift reply.

The checksum for the Roboto-Thin.ttf is 5ecbc99d1a81fed7dc71cb068ec0a74d

It's the same checksum on both localhost and on our production server. I have zipped the font and attached it.

[Roboto-Thin.ttf.zip](https://github.com/pcwalton/font-kit/files/3385845/Roboto-Thin.ttf.zip)

**Comment by RazrFalcon** [comment]
Are you using the wasm mode? Because on Linux it works correctly.

**Comment by DHTMLGoodies** [comment]
I apologize - my mistake. This is an issue with the javascript version of fontkit. This is not the repository for that.

**Comment by RazrFalcon** [comment]
No problem =)
